lib/active_admin/helpers/collection.rb

```diff
@@ -3,13 +3,13 @@ module Helpers
     module Collection
       # 1. removes `select` and `order` to prevent invalid SQL
       # 2. correctly handles the Hash returned when `group by` is used
-      def collection_size(c = collection)
+      def collection_size(c = collection())
```

Comment:
Right, it works either. What I actually fixing here are warnings like [these](https://github.com/rails/rails/pull/18160/files). Actually activeadmin doesn't work on Ruby >= 2.2.0 without this fix at all.